sportsdataverse/pwhl/__init__.py

```
@@ -6,3 +6,25 @@
 from sportsdataverse.pwhl.pwhl_analytics import *  # noqa: F401,F403
 from sportsdataverse.pwhl.pwhl_loaders import *
 from sportsdataverse.pwhl.pwhl_loaders_extra import *
+from sportsdataverse.pwhl.pwhl_prediction_constants import (
```

Flake8 (7.3.0)

[error] 'sportsdataverse.pwhl.pwhl_prediction_constants.LEAGUE_CONSTANTS' imported but unused

(F401)